README.md

```diff
@@ -11,6 +11,8 @@
 
 # RosettaSciIO
 
+<img src="https://github.com/jordiferrero/rosettasciio/tree/main/docs/_static/logo_rec_oct22.svg" width="300" alt="RosettaSciIO">
```

Comment:
Maybe this should point to https://github.com/hyperspy/rosettasciio ?

Reply:
Well spotted :-)